diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9fde713..f40dfa9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -18,7 +18,7 @@ IF( ${CMAKE_COMPILER_IS_GNUCXX} )
 ENDIF( ${CMAKE_COMPILER_IS_GNUCXX} )
 endif(${WIN32})
 
-option(USE_FTD2XX "Use FTDI libFTD2XX instead of free libftdi" ON)
+option(USE_FTD2XX "Use FTDI libFTD2XX instead of free libftdi" OFF)
 
 pkg_check_modules(LIBFTDI REQUIRED libftdi1)
 include_directories(${LIBFTDI_INCLUDE_DIRS})
